### PR TITLE
Add Intro, QuickStart, FAQ, Command Summary, Table of Contents to UG

### PR DESCRIPTION
**Pull request by matheril**
Adds information to the User Guide for v1.0.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -2,10 +2,9 @@
 
 ## Introduction
 
-SplitLah is an application for keeping track of expenses during group outing sessions, specifically for those who are 
-budget conscious. It then proceeds to split the expenses according to the group members' individual costs for the 
-activities they participated in, for a particular session, via a Command Line Interface(CLI).
-
+SplitLah is an application for people who split bills equally amongst themselves after a group outing. SplitLah does
+the work of calculating how much each participant must pay across multiple activities, and keeps track of who has paid
+and who has not. You can use SplitLah through a Command Line Interface (CLI).
```
Comment (warrencxw) [inline docs/UserGuide.md:7]: The keeping track of who has paid and not is not part of v1.0, unless you mean keep track of who paid for what activities? Maybe consider rephrasing in that case to be clearer.

**Review by froststein — CHANGES_REQUESTED** [review]
Perhaps can add in table of contents

**Review by warrencxw — CHANGES_REQUESTED** [review]
Additions look good. However, Introduction has a very ambiguous statement which may imply that we are providing something that we don't, and it has to be fixed.

File: docs/UserGuide.md (hunk 1 above)
Comment (matheril) [reply docs/UserGuide.md]: Resolved

**Review by warrencxw — APPROVED** [review]
Looks good to merge.

**Review by froststein — APPROVED** [review]

**Pull request merged**